web-ng/src/app/services/data-store/data-store.service.ts

```diff
@@ -75,6 +76,25 @@ export class DataStoreService {
       });
   }
 
+  async deleteLayer(projectId: string, layerId: string) {
+    await this._deleteAllFeaturesInLayer(projectId, layerId);
+    return await this.db
+      .collection('projects')
+      .doc(projectId)
+      .update({
+        [`layers.${layerId}`]: firebase.firestore.FieldValue.delete(),
+      });
+  }
+
+  async _deleteAllFeaturesInLayer(projectId: string, layerId: string) {
```

Comment:
As a matter of convention we haven't been using `_` prefix or suffix for private methods; instead, please use the `private` keyword.

Reply:
Done.